OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+# Oracle WebCenter Sites 12c R2 (12.2.1.3.0) on Docker
+*This page is best viewed on Mozilla Firefox and Google Chrome browsers.*
+
+## Contents
+
+### 1. [Introduction](#1-introduction-1)
+### 2. [Hardware and Software Requirements](#2-hardware-and-software-requirements-1) 
+### 3. [Prerequisites](#3-prerequisites-1)
+### 4. [Downloading Docker Images and Oracle WebCenter Sites Binary](#4-downloading-docker-images-and-oracle-webcenter-sites-binary-1)
+### 5. [Building Oracle WebCenter Sites Docker Images](#5-building-oracle-webcenter-sites-docker-images-1)
+### 6. [Preparing to run Oracle WebCenter Sites Docker Image](#6-preparing-to-run-oracle-webcenter-sites-docker-image-1)
+### 7. [Running Oracle WebCenter Sites Docker Image](#7-running-oracle-webcenter-sites-docker-image-1)
+### 8. [FAQs](#8-faqs-1)
+### 9. [Copyright](#9-copyright-1)
+
+## 1. Introduction
+To create web content management solutions, developers need a lightweight environment. Docker images need minimum resources, thereby allowing developers to quickly create development environments.
+
+This project offers scripts to build an Oracle WebCenter Sites image based on 12c R2 (12.2.1.3). Use this Docker configuration to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Sites, see the [Oracle WebCenter Sites Online Documentation](https://docs.oracle.com/middleware/12213/wcs/index.html).
+
+This project creates Oracle WebCenter Sites Docker image with a single node targeted for development and excludes components such as SatelliteServer, SiteCapture, and VisitorServices. This image is not supported or recommended to use on a production system.
+
+## 2. Hardware and Software Requirements
+Oracle WebCenter Sites on Docker is certified to run on hardware and software listed below:
+
+### A. Hardware Requirements
+
+    | Hardware  | Size  |
+    | RAM       | 16GB  |
+    | Disk Space| 200GB+|
+
+### B. Software Requirements
+
+    |       | Version                                   | Command to verify version |
+    | OS    | Oracle Linux Server release 7.3           | more /etc/oracle-release  |
+    | Docker| Docker version 17.03.1-ce, build 5cda16c  | docker –version           |
```

Comment:
Please specify 17.03 or higher. The current stable release for Oracle Linux 7 is 17.06, for example.

Reply:
Changes incorporated into latest file.